app/src/main/java/org/systers/mentorship/view/fragments/EditProfileFragment.kt

```diff
@@ -97,4 +105,20 @@ class EditProfileFragment: DialogFragment() {
         profileViewModel.successfulUpdate.removeObservers(activity!!)
         profileViewModel.successfulUpdate.value = null
     }
+
+    /**
+     * A function to perform input validation
```

Comment:
This is too much broken into lines, you have 8 lines that could be better distributed right?

Reply:
Should we do here length validations how it is on the backend? @m-murad

Reply:
@isabelcosta Thanks for the suggestion. I have made the this change. The comments now have longer lines. I'm also adding length validations, just to be safe (yay for the modular system!)